### DISCUSSION TITLE
Edge with Native Postgres database.

### DISCUSSION THREAD

**Discussion by rkumar0322** [geldata/gel#2714] [Q&A]
I am looking to reuse my old instance of postgres with edgedb. How would it work in this case. Is this possible?

**Comment by kafein** [comment]
I guess using postgres instance with your existing schema is not possible, because EdgeDB structures the data differently. You may want to re-structure your database schema with the EdgeDB, then you can write some migration scripts to migrate your existing data to EdgeDB. 

You can check @elprans comment;
https://github.com/edgedb/edgedb/discussions/2674#discussioncomment-1001348
